docs/UserGuide.md

```diff
@@ -39,136 +39,281 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
-</div>
-
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-
-### Adding a person: `add`
-
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
-
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
-
-### Listing all persons : `list`
-
-Shows a list of all persons in the address book.
-
-Format: `list`
-
-### Editing a person : `edit`
-
-Edits an existing person in the address book.
+---
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+### Feature 2: Add New Customer
+
+**Purpose:**  
+This feature allows you to enter and save detailed records for new customers. Each customer's record includes their name, contact number, email, occupation, and income.
+
+**How to Use It:**  
+- **Command Format:** 
+```
+add --name <customer name> --contact <customer contact> --email <customer email> --job <job name> --income <customer income>
+ ```
+- **Example:** 
+```
+add --name TAN LESHEW --contact 66997788 --email mrtan@ntu.com --job doctor of computer science --income 1000000000
+```
+
+#### Parameters
+| Parameter | Expected Format                              | Explanation                                                |
+|-----------|----------------------------------------------|------------------------------------------------------------|
+| name      | Alphanumeric, capitalized                    | Names are in block letters for clarity and consistency.    |
+| contact   | 8-digit number, starts with 8 or 9           | Ensures the contact number is valid in Singapore.          |
+| email     | Must include "@" and domain, case insensitive| Verifies that the email address is in a standard format.   |
+| job       | Any text, case insensitive                   | Accepts all job titles without case sensitivity.           |
+| income    | Non-negative integers                        | Only positive numbers or zero are valid for income fields. |
+
+#### What to Expect
+- **If Successful:** You'll see a message: "New Contact added. Their ID is `<new customer ID>`."
+- **If There is an Error:** 
+  - Missing name: "Customer requires a name."
+  - Incorrect email format: "Please give a valid email address."
+  - Invalid income entry: "Please use a non-negative number for income."
+
+**Handling Duplicates:**  
+If a customer with the same name, email, job, and income is already saved, you'll get a message: "This customer is already saved as a contact. Their ID is `<new customer ID>`."
```

Comment:
Same thing here, probably remove the part with the customer id. "If a customer with the same name, email, job, and income is already saved, you'll get a message: "This customer is already saved as a contact." should suffice.